docs/lib/datastore/fragments/native_common/datastore-events.md

```diff
@@ -1,14 +1,102 @@
 Amplify uses `Hub` for different categories to communicate with one another when specific events occur. DataStore emits `Hub` messages for the following events:
 
+## networkStatus
+
+Dispatched when DataStore starts and every time network status changes
+
+HubPayload `NetworkStatusEvent` contains a boolean value `active` to notify network status
+```
+{ active: true }
+```
+
+## subscriptionsEstablished
+
+Dispatched when DataStore has finished establishing its subscriptions to all sync-able models
```

Comment:
what is "sync-able" from a customer perspective? I think we can omit it.

Reply:
removed